### PR TITLE
Check connection fingerprint with mbedtls

### PR DESCRIPTION
**Pull request by gasperov**

### CODE REVIEW

**Review by paullouisageneau — COMMENTED** [review]
Thank you for adding the verification, I just have a few comments.

File: src/impl/dtlstransport.cpp [hunk 1]
```diff
@@ -400,6 +400,26 @@ DtlsTransport::DtlsTransport(shared_ptr<IceTransport> lower, certificate_ptr cer
...
+				char crt_info[1024];
+				char verify_info[64];
+				crt_info[0] = verify_info[0] = 0;
+				mbedtls_x509_crt_info(crt_info, sizeof(crt_info), "", crt);
+				if ((*flags) == 0) {
+					strcpy(verify_info, " - no flags");
+				} else {
+					mbedtls_x509_crt_verify_info(verify_info, sizeof(verify_info), "  ! ", *flags);
+				}
+				PLOG_DEBUG << "Verify requested for (Depth " << depth << "):" << crt_info << verify_info;
```
[the remark this answers is not in the document]
Comment (gasperov) [reply src/impl/dtlstransport.cpp:416]: will remove logs ...

**Comment by gasperov** [comment]
I am not sure what is wrong with OpenSSL test:
> 2023-06-11 13:14:49.322 ERROR [4640] [rtc::impl::TlsTransport::doRecv@758] TLS recv: OpenSSL error: error:00000001:lib(0):func(0):reason(1)

**Review by paullouisageneau — APPROVED** [review]
It looks good, thank you!

**Pull request merged**